src/common/constants.js

```diff
@@ -36,5 +41,13 @@ module.exports = {
     5: 'payoneer',
     6: 'western union',
     7: 'wipro payroll'
+  },
+
+  // The mapping between the profile completion metadata field name and the traits id
+  // The keys are the trait ids, the values are the corresponding field in the metadata object
+  TRAITS_TO_PROFILE_COMPLETION_CHECKLIST_METADATA_MAP: {
```

Comment:
@Schpotsky does it need more mappings for skills, profile_picture, bio etc? or Am I reading it wrong?

Reply:
I think I got it. The other fields that I mentioned are not determined by traits, rather they are either direct property of the member model or entirely different model (skills).